tests/integration/roleTemplate_test.go

```diff
@@ -118,3 +121,32 @@ func (m *IntegrationSuite) TestRoleTemplateNoAPIGroups() {
 	}
 	validateEndpoints(m.T(), endPoints, m.clientFactory)
 }
+
+// ProjectCreatorDefault=true should not have other than Project context
+func (m *IntegrationSuite) TestRoleTemplateWithProjectCreatorDefault() {
+	invalidRoleTemplate := &v3.RoleTemplate{
+		ObjectMeta: v1.ObjectMeta{
+			Name: "test-roletemplate-invalid-context",
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				Verbs:     []string{"GET", "WATCH"},
+				APIGroups: []string{"v1"},
+				Resources: []string{"pods"},
+			},
+		},
+		Context:               "cluster",
+		ProjectCreatorDefault: true,
+	}
+	gvk, err := m.clientFactory.GVKForObject(invalidRoleTemplate)
+	m.Require().NoError(err, "Failed to get gvk")
+	client, err := m.clientFactory.ForKind(gvk)
+	m.Require().NoError(err, "Failed to create client")
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
+
+	err = client.Create(ctx, invalidRoleTemplate.GetNamespace(), invalidRoleTemplate, nil, v1.CreateOptions{})
+	m.Assert().NotNil(err)
+	m.Assert().Contains(err.Error(), "RoleTemplate context must be Project when projectCreatorDefault=true")
```

Comment:
I don't think this error is right?
```suggestion
	m.Assert().Contains(err.Error(), "RoleTemplate context must be project when projectCreatorDefault=true")
```

Reply:
Fixed